mule-packager/src/main/java/org/mule/tools/api/util/exclude/GlobMatcherFileReader.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.util.exclude;
+
+import com.beust.jcommander.internal.Lists;
+import com.google.common.io.LineProcessor;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.apache.commons.lang3.StringUtils.startsWith;
+import static org.apache.commons.lang3.StringUtils.strip;
+
+class GlobMatcherFileReader implements LineProcessor<List<GlobMatcher>> {
+
+  private List<GlobMatcher> globMatchers = Lists.newArrayList();
```

Comment:
please do not uses this kind of classes Lists.newArrayList() 
Its better to just do new ArrayList() which is what this maven class does under the hood 
-> 

>     public static <K> List<K> newArrayList() {
>         return new ArrayList();
>     }

If needed please use java native classes apache or google libraries for this kind of util methods .
In this commit you also used  com.google.common.collect.Lists.newArrayList I prefer you use that if you don't like new ArrayList()

Reply:
Ok!